components/media-feature/Component.jsx

```diff
@@ -39,49 +40,49 @@ export default function MediaFeature({
   return (
     <Container width="full" paddingX={false}>
       <section className="su-py-45 su-px-20 su-flex su-justify-center su-relative md:su-py-72 md:su-px-50">
-        <div className="su-max-w-[1086px] su-flex su-flex-col su-items-center su-z-[2] su-relative su-p-38 before:su-content-[''] before:su-bg-foggy-light before:su-w-full before:su-h-full before:su-opacity-90 before:su-absolute before:su-z-[-1] before:su-top-0 before:su-left-0 md:su-flex-row md:su-gap-20 md:su-items-start lg:su-p-48 lg:su-gap-48">
-          {/* <div className="su-h-[224px] su-w-[224px] su-relative su-shrink-0 md:su-w-[182px] md:su-h-[182px] lg:su-w-[292px] lg:su-h-[292px]"> */}
-          <div
-            className={`${
-              mediaType === "Podcast" &&
-              "su-w-[224px] md:su-w-[182px] lg:su-w-[292px]"
-            } su-h-[224px] md:su-h-[182px] lg:su-h-[292px] su-relative su-shrink-0 ${
-              mediaType === "Book" && "media-feature__thumb"
-            }`}
-          >
-            <img
-              // src="https://picsum.photos/600/250"
-              src={imageData.url}
-              alt={imageData.attributes.alt}
-              className={thumbMap.get(mediaType)}
-              // className="su-media-card-thumb su-size-full su-object-scale-down su-object-center"
-              // className="su-absolute su-object-cover su-rounded-[8px] su-z-[2] su-flex-1 su-size-full su-shadow-[0px_4px_7px_0px_rgba(0,0,0,0.15)]"
-            />
-          </div>
-
-          <div>
-            <div className="su-py-20 su-w-full md:su-pb-27 md:su-pt-0 *:dark:su-text-black">
-              <FeaturedHeading type={mediaType} />
+        <a href={linkUrl} className="su-media-feature-title-link su-group">
```

Comment:
Place the `<a>` inside the `<h3>` below and add the `su-stretched-link` class – you can see this pattern in packages > card > verticalCard.jsx as reference. This keeps the link markup semantic while stretching the link across the container. (`su-relative` required on container div for this to work properly, see comment above)

Reply:
Remove `su-group` from this `<a>` element, add to container. See comment above.

Reply:
Oh, `su-stretched-link` is so much mo' bettah. Thank you!